### PR TITLE
SDK: Product Lifecycle refinement, including titles for GitHub releases

### PR DESCRIPTION
**Pull request by QuintinWillison**
This came up when discussing [the title for the `v1.1.0-rc.1` release of Ably Asset Tracking for Android](https://github.com/ably/ably-asset-tracking-android/releases/tag/v1.1.0-rc.1) with @KacperKluka.

### CODE REVIEW

**Comment by QuintinWillison** [comment]
Oops, I missed a heading. One more commit to come, sorry...

File: sdk/product-lifecycle.md [hunk 1]
```diff
@@ -4,39 +4,49 @@ Two preliminary lifecycle stages - Alpha and Beta - are designed to ensure that
 
 See [Semantic Versioning (SemVer)](https://semver.org/) in respect of the terminology used in this document, specifically "Pre-Release Version Suffix".
 
-## Labs
+The Pre-Release `<build>` component should always be a positive integer value.
```
Comment (KacperKluka) [inline sdk/product-lifecycle.md:7]: Shouldn't this be under the `## Pre-Releases`? :wink:

Comment (QuintinWillison) [reply sdk/product-lifecycle.md:7]: My logic was that I was grouping it with the previous para relating to SemVer, but your point is very valid. I shall move it. 👍

Comment (QuintinWillison) [reply sdk/product-lifecycle.md]: https://github.com/ably/engineering/pull/20/commits/f11c97856714c5ca95ec79ebba339c5e23c140aa

**Review by KacperKluka — APPROVED** [review]
Looks good :grin:

**Review by AndyNicks — APPROVED** [review]

**Review by stmoreau — APPROVED** [review]

**Pull request merged**